### PR TITLE
Added AngularJS course in English by My Lesson

### PR DESCRIPTION
**Pull request by prerna120**
## What does this PR do?
Added AngularJS course in english

### Description
Youtube Video: https://www.youtube.com/watch?v=9b9pLgaSQuI

### Why is this valuable (or not)?
It is one video course covering all the concepts of angularJS

### How do we know it's really free?
It's on youtube.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -665,6 +665,7 @@
 
 > :information_source: See also &#8230; [Angular](#angular)
 
+* [AngularJS Tutorial for Beginners](https://www.youtube.com/watch?v=9b9pLgaSQuI) - Yaakov Chaikin, My Lesson (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:668]:
```suggestion
* [AngularJS Tutorial for Beginners](https://www.youtube.com/watch?v=9b9pLgaSQuI) - Yaakov Chaikin, (My Lesson)
```

Comment (prerna120) [reply courses/free-courses-en.md]: Done suggested change

**Comment by prerna120** [comment]
Hi @eshellman , please review my pull request.

**Comment by eshellman** [comment]
sorry for the delay. see #9109

**Pull request merged**